### PR TITLE
docs: update guidelines for duplicates

### PR DESCRIPTION
**Pull request by EricFromCanada**
As per #2873, duplicate and versioned formulae are fine if done right.

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [N/A] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [N/A] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: docs/brew-tap.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ but the command isn't limited to any one location.
 
 ```sh
 $ brew tap
-homebrew/dupes
+homebrew/core
```
Comment (MikeMcQuaid) [inline docs/brew-tap.md]: Maybe `homebrew/bundle` as `homebrew/core` is tapped by default?

Comment (EricFromCanada) [inline docs/brew-tap.md]: My logic was to mirror its actual output, but ¯\_(ツ)_/¯

Comment (MikeMcQuaid) [inline docs/brew-tap.md]: @EricFromCanada I'm an idiot. Changed it back. Sorry!!

**Comment by MikeMcQuaid** [comment]
Thanks again @EricFromCanada and apologies for my earlier misreading.

**Pull request merged**